### DISCUSSION TITLE
Welcome to icehrm Discussions!

### DISCUSSION THREAD

**Discussion by gamonoid** [gamonoid/icehrm#238] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by d76g** [comment]
I am facing a problem in IceHRM Installation page, I filled my information and the Install Application button does not turn to be clickable.
